### PR TITLE
[doc] update fork/doc.go

### PR DESCRIPTION
**Pull request by laggu**
## Proposed changes

- update fork/doc.go

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fork/doc.go [hunk 1]
```diff
@@ -14,5 +14,13 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package fork has information related to hard forks.
+/*
+Package fork has information related to hard forks.
+
+HardForkConfigs (blockNumber of hard fork) will be declared in this package.
```
Comment (aidan-kwon) [inline fork/doc.go]:
```suggestion
HardForkConfig which has hard fork blockNumbers is declared in this package.
```

File: fork/doc.go [hunk 2]
```diff
@@ -14,5 +14,13 @@
...
+
+Source Files
+
+- fork.go : Declare HardForkConfigs
```
Comment (aidan-kwon) [inline fork/doc.go]:
```suggestion
- fork.go : Declare HardForkConfig
```

File: fork/doc.go (hunk 1 above)
Comment (laggu) [reply fork/doc.go]: reflected!

File: fork/doc.go (hunk 2 above)
Comment (laggu) [reply fork/doc.go]: reflected!

**Pull request merged**